channels/gr.m3u

```diff
@@ -169,8 +167,6 @@ http://94.130.180.175:8081/live/greektv/playlist.m3u8
 https://vdo.alphaserver.gr:3305/stream/play.m3u8
 #EXTINF:-1 tvg-id="NEATV.gr" tvg-name="NEA TV" tvg-country="GR" tvg-language="" tvg-logo="http://greektv.pbworks.com/w/file/fetch/46926622/NEA_TV.png" group-title="",NEA TV (1080p)
 https://live.neatv.gr:8888/hls/neatv.m3u8
-#EXTINF:-1 tvg-id="Nea TV GR" tvg-name="Nea TV GR" tvg-country="GR" tvg-language="Modern Greek (1453-)" tvg-logo="http://greektv.pbworks.com/w/file/fetch/46926622/NEA_TV.png" group-title="Local",Nea TV
```

Comment:
Basically replace this one by this one : 

https://live.neatv.gr:8888/hls/neatv_mid/index.m3u8

Reply:
Replaced